src/components/20-molecules/datepicker/react/story.jsx

```diff
@@ -38,6 +38,7 @@ storiesOf('Molecules/Datepicker/React', module)
         labelbuttoncancel="Cancel"
         labelbuttonok="OK"
         allowedyears={[2019, 2020]}
+        // eslint-disable-next-line no-console
```

Comment:
I don't like console.log statements in stories, since a user probably does not have the console open when he is testing our demos. IMHO 'alert' is the least we should provide, but I even better like a div containing some state text, that changes upon events that happen.

Reply:
Deleted since underlying change callback was problematic in triggering behaviour - thanks for spotting this!